rsync_to_rdisc.py

```diff
@@ -1,51 +1,46 @@
-#! /usr/bin/env python
+#! /usr/bin/env python3
 import sys
 import os
-import commands
 import datetime
-from pwd import getpwuid
-from os.path import join, isfile, split
+
 from email import encoders
 from email.mime.multipart import MIMEMultipart
 from email.mime.base import MIMEBase
 from email.mime.text import MIMEText
 import smtplib
 import mimetypes
 import socket
-import settings
-
-####################################################################################################################################
-# Purpose:    Automatic sync Dx-run folders to bgarray
-####################################################################################################################################
+from paramiko import SSHClient
+from pathlib import Path
 
-def find_owner(filename):
-    return getpwuid(os.lstat(filename).st_uid).pw_name
+import settings
 
 
-def make_mail(filename, state):
-    if state[0] == "lost":
-        email_to = [settings.owner_dic[item] for item in settings.owner_dic]
+def make_mail(filename, state, reason=None, run_file=None):
+    if state == "lost_mount":
         subject = "ERROR: mount lost to BGarray for {}".format(socket.gethostname())
-        text = "<html><body><p>Mount to BGarray is lost for {}</p></body></html>".format(socket.gethostname())
-    else:
-        if state[0] == "ok":
-            subject = "COMPLETED: Transfer to BGarray has succesfully completed for {}".format(filename)
-            text = "<html><body><p>Transfer to BGarray has succesfully completed for {}</p></body></html>".format(filename)
-        elif state[0] == "error":
-            subject = "ERROR: transfer to BGarray has not completed for {}".format(filename)
-            text = "<html><body><p>Transfer to BGarray has not been completed for {}</p></body></html>".format(filename)
-        elif state[0] == "notcomplete":
-            subject = "Exome analysis not complete (no workflow.done file). Run = {}!".format(filename)
-            text = "<html><body><p> Data not transferred to BGarray. Run {}</p></body></html>".format(filename)
-
-        file_owner = find_owner(filename)
-        if file_owner not in settings.owner_dic:
-            email_to = [settings.finished_mail]
-            subject = "WARNING user {} does not exist! {}".format(file_owner, subject)
-        else:
-            email_to = [settings.owner_dic[file_owner]]
-
-    send_email(settings.email_from, email_to, subject, text)
+        text = ("<html><body><p>Mount to BGarray is lost for {0}</p>"
+                "<p>Remove {1} before datatransfer can be restarted</p></body></html>".format(socket.gethostname(), run_file))
```

Comment:
I prefer
```python
text = (
    "<html><body><p>Mount to BGarray is lost for {0}</p>"
    "<p>Remove {1} before datatransfer can be restarted</p></body></html>"
).format(socket.gethostname(), run_file)
```

You also have it styled like this on other lines, for example, line 90: